spec/system/candidate_entering_qualification_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'A candidate adding a qualification' do
+  include TestHelpers::NonDegreeQualifications
+
+  context 'who successfully enters their details' do
+    before do
+      visit '/qualifications/new'
+
+      fill_in_qualification_details
+
+      click_on t('application_form.save_and_continue')
+    end
+
+    it 'sees a summary of those details' do
+      visit '/check_your_answers'
+
+      expect(page).to have_content('GCSE')
+    end
+
+    it 'can go back and edit them' do
+      visit '/check_your_answers'
+
+      find('#change-qualification').click
+      expect(page).to have_field('Type of qualification', with: 'GCSE')
+    end
```

Comment:
Do we test the flow of adding another qualification anywhere?

Reply:
I did not get the question, here we are testing that
the user 
from the check-your-answers page,
can go back and edit the qualification

Reply:
https://trello.com/c/qo5UkCSw/697-use-docker-in-local-development-to-match-production-environment

Reply:
ok, I'll add a test for it